gendst/data/positions.go

```diff
@@ -280,7 +280,12 @@ A /*Label*/ : /*Colon*/
 
 	// TypeSpec(1)
 	type (
-		/*Start*/ T2 = /*Name*/ T1 /*End*/
+		/*Start*/ T2 = /*AliasName*/ T1 /*End*/
```

Comment:
Was it intentional to rename `Name` to `AliasName`? This would be a breaking change...

Reply:
Unfortunately, it *was* intentional. I didn't necessarily realize that it would be a breaking change. Here's the problem:

```
type (
/*start*/ T[P any] /*Name*/ []P
)
```
is the only way decorations will work in that case because the `Name` decoration has to come after the set of type parameters because the `Name` decoration goes after the `=` in an AliasDecl. 

In other words, we cannot do this:
```type (
/*Start*/ T /*Name*/ [P any] /*TypeParams*/ []P
)
```

I am not 100% sure whether that is something you want to support, but that is where the `AliasName` came from -- a way to differentiate the decorator. 

I am sorry if I am not explaining clearly. Please let me know if I confused the situation! I am happy to resolve it however you deem appropriate -- you are the expert!

Reply:
What do you think about this?

```go
	// TypeSpec(0)
	type (
		/*Start*/ T1 /*Name*/ []int /*End*/
	)

	// TypeSpec(1)
	type (
		/*Start*/ T2 = /*Name*/ T1 /*End*/
	)

	// TypeSpec(2)
	type (
		/*Start*/ T3 /*TypeParams*/ [P any] = /*Name*/ []P /*End*/
	)
```

Reply:
Would that work? I imagine putting a comment between the name and the type parameters will be pretty rare...

Reply:
Interestingly enough, `TypeSpec(2)` is invalid because it is not possible to have a generic alias. So, that might not be a problem at all! 

This, however, does not work:

```
type (
/*Start*/ T3 /*TypeParams*/ [P any] /*Name*/ []P /*End*/
)
```

Reply:
OK cool that works. I haven't actually played with Go generics on a project yet 😂

Reply:
Oh sorry I read "This, however, does work:".... how doesn't it work?

Reply:
No problem! It doesn't work because it wants /*End*/ after T3 because it considers that the _end_ of an `Ident`. 

Sorry for the delay responding!

Reply:
Hmmm I think if that's the problem, then we have something wrong in the setup... there will be an ident in there which will have start and end annotation, but the additional annotation should work just fine... take a look how similar nodes work... 🤔

Reply:
e.g. take a look at CallExpr.Fun - that's the same scenario, right?

Reply:
Your suggestion is perfect!!